suite-native/atoms/src/HStack.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode } from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { NativeSpacing } from '@trezor/theme';
+
+import { Box, BoxProps } from './Box';
+
+interface HStackProps extends BoxProps {
+    children: ReactNode;
+    spacing?: NativeSpacing | number;
+}
+
+type SpacerStyleProps = {
+    isLastChild: boolean;
+    spacing?: NativeSpacing | number;
+};
+
+const spacerStyle = prepareNativeStyle<SpacerStyleProps>((utils, { isLastChild, spacing }) => {
+    const spacingValue = typeof spacing === 'number' ? spacing : utils.spacings[spacing ?? 'small'];
+    return {
+        extend: {
+            condition: !isLastChild,
+            style: {
+                marginRight: spacingValue,
+            },
+        },
+    };
+});
+
+export const HStack = ({ children, style, spacing, ...rest }: HStackProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Box flexDirection="row" style={style} {...rest}>
```

Comment:
is this flexDirection only one diference between HStack and VStack? That looks little bit redundant to me and we nice to unify it. Maybe create just `Stack`?

Reply:
Yeah, I wanted to do this in next PR to create better component. But didn't want to expand scope of this one. I will create task for it and want to give this component more thought and make it more reusable. I will link the task here so you can consider it resolved.

Reply:
Agree, please create task for it.